src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -156,12 +157,23 @@ public void setGenotypingErrorRate(final double genotypingErrorRate) {
     }
 
     /**
-     * Sets the maximal difference in PL scores considered when reading PLs from a VCF.
+     * Does same thing as {@link #setMaximalPLDifference(int)} but named in the compliant way.
+     *
+     * @deprecated use {@link #setMaximalPLDifference(int)} instead.
+     * @since 16/4/2020
      */
+    @Deprecated
     public void setmaximalPLDifference(final int maximalPLDifference) {
-        this.maximalPLDifference = maximalPLDifference;
+        setMaximalPLDifference(maximalPLDifference);
     }
 
+    /**
+     * Sets the maximal difference in PL scores considered when reading PLs from a VCF.
+     * @deprecated since 04/2020. use CappedHaplotypeProbabilities and explicitly cap your probabilities.
+     */
+    @Deprecated
+    public void setMaximalPLDifference(final int maximalPLDifference) { }
```

Comment:
Why add a new method name with no contents that is deprecated?

Reply:
evolution

Reply:
I'll just remove these as they don't do anything....better to have a compile error than a slient noop